src/components/dialog/dialog.js

```diff
@@ -272,8 +272,8 @@ function MdDialogProvider($$interimElementProvider) {
   function advancedDialogOptions($mdDialog) {
     return {
       template: [
-        '<md-dialog aria-label="{{ dialog.ariaLabel }}">',
-          '<md-content>',
+        '<md-dialog aria-label="{{ dialog.ariaLabel }}" tabIndex="-1">',
+          '<md-content role="document" tabIndex="0">',
```

Comment:
I thought it was applied to all dialogs. I didn't realize it was only applied to `alert` and `confirm` dialogs.
Sorry for the noise :)